front/app/containers/Admin/settings/messages.ts

```diff
@@ -133,6 +133,16 @@ export default defineMessages({
     defaultMessage:
       'The provided subtitle exceeds the maximum allowed character limit',
   },
+  bannerDisplayHeaderAvatars: {
+    id: 'app.containers.AdminPage.SettingsPage.bannerDisplayHeaderAvatars',
+    defaultMessage: 'Display avatars',
+  },
+  bannerDisplayHeaderAvatarsSubtitle: {
+    id:
+      'app.containers.AdminPage.SettingsPage.bannerDisplayHeaderAvatarsSubtitle',
+    defaultMessage:
+      'Show profile pictures of participats and number of them to non-registered visitors',
```

Comment:
Nice catch, thanks.

Reply:
> I wouldn't write "non-registered users" because in i6 this will also be the banner for signed-in users.

I assumed it's related only to text. I'll ask Karel anyway.